upup/pkg/fi/cloudup/awstasks/route.go

```diff
@@ -155,6 +155,15 @@ func (_ *Route) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Route) error {
 		} else if e.InternetGateway != nil {
 			request.GatewayId = checkNotNil(e.InternetGateway.ID)
 		} else if e.NatGateway != nil {
+			// It takes 'forever' (up to 5 min...) for a NatGateway to become available after it has been created
+			// We have to wait untill it is actualy up
+			params := &ec2.DescribeNatGatewaysInput{
+				NatGatewayIds: []*string{e.NatGateway.ID},
+			}
+			err := t.Cloud.EC2().WaitUntilNatGatewayAvailable(params)
```

Comment:
I don't think it's a failure - as kops will wait for it to come up - this just offers stability, and cleaner logs, and prevents a potential headache later if anyone tries to code in support for the natgateway without waiting for it to come up..